pom.xml

```diff
@@ -12,13 +12,14 @@
 
 	<groupId>io.disc99</groupId>
 	<artifactId>spring-cloud-stream-binder-kinesis-legacy-parent</artifactId>
-	<version>0.9.1.RELEASE</version>
+	<version>0.9.3.RELEASE</version>
 	<packaging>pom</packaging>
 
 	<properties>
 		<spring-cloud-stream.version>1.3.0.RELEASE</spring-cloud-stream.version>
 		<spring-cloud-aws.version>1.2.2.RELEASE</spring-cloud-aws.version>
 		<spring-integration-aws-version>1.1.0.RELEASE</spring-integration-aws-version>
+		<spring-integration-core-version>4.3.12.RELEASE</spring-integration-core-version>
```

Comment:
Latest `4.3.x` version is `4.3.21` but in some cases such as spring boot `1.x` old version (e.g. `1.3.3` ) it does not work, so specify stable version `4.3.12` 😭